packages/bun-usockets/src/bsd.c

```diff
@@ -905,7 +905,7 @@ static int bsd_do_connect(struct addrinfo *rp, int *fd)
             return LIBUS_SOCKET_ERROR;
         }
 
-        int resultFd = bsd_create_socket(rp->ai_family, rp->ai_socktype, rp->ai_protocol);
+        int resultFd = bsd_create_socket(rp->ai_family, SOCK_STREAM, 0);
```

Comment:
why hardcode it?

Reply:
these are the only valid values for a TCP socket, and makes `bsd_do_connect` independent of the values returned by `getaddrinfo` beyond the resolved addresses